nix/runtime.nix

```diff
@@ -0,0 +1,37 @@
+{
+  # Plumbing tools:
+  closureInfo, runCommand, writeText,
+
+  # Actual dependencies to propagate:
+  bash, coreutils }:
+let
+  runtimePaths = [ coreutils bash ];
+
+  runtimeClosureInfo = closureInfo {
+    rootPaths = runtimePaths;
+  };
+
+  runtimeClosureInfoAsNix = runCommand "runtime-closure.nix" {}
+  ''
+    cat <<EOF > scratch.nix
+    # Nix with sandboxing requires every path used at build time be
+    # explicitly declared. If we simply passed in the paths, they
+    # would be copied in as sources. Using builtins.storePath we're
+    # able to tell Nix that, no, in fact, treat these not as sources
+    # to copy, but instead of a regular store path.
+    #
+    # Include the explicit closure, too, otherwise we'll get mysterious
+    # "file not found" errors due to the glibc interpreter being
+    # missing.
+    {
```

Comment:
storePath only works if we're in a /nix/store store, not if we're in the `nix-build` fake store. 

This is very annoying.

Nix has some magic in its `<nix/config.nix>` but I'm not 100% sure that is the right thing to do.

Generally frustrated.